packages/react/src/ActionMenu/ActionMenu.examples.stories.tsx

```diff
@@ -410,15 +410,15 @@ export const MultipleSections = () => {
       </ActionMenu.Anchor>
       <ActionMenu.Overlay width="small">
         <ActionList>
-          <ActionList.Group selectionVariant="multiple">
+          <ActionList.Group>
```

Comment:
Fixes: `Required ARIA attributes must be provided` for [Multiple Sections story](https://primer.style/react/storybook/?path=/story/components-actionmenu-examples--multiple-sections)